### ISSUE TITLE
AssemblyScript Public Meeting #2 - December 4th, 2019

### ISSUE THREAD

**Issue by torch2424** [AssemblyScript/meta#16]
# Date and Time

This public meeting will take place: December 4th, 2019, 19:00 UTC (11:00 AM US Pacific Standard Time UTC -8)

# General Agenda

* Agenda Items from this Github issue
* Team Updates
  * Compiler
  * Tooling
  * Advocacy
* Additional in-meeting comments / discussion

Feel free to comment on this issue if you have any agenda items you would like to bring up. 
Meeting Notes will be placed on this issue for those who cannot make the meeting.

# Meeting Information

We will use Google Meet for our meetings. Our meeting room is: https://meet.google.com/ofw-zkpi-aek

Anyone who is contributing to the AssemblyScript project, building something with AssemblyScript, interested in the Assembly project or WebAssembly in general, is welcome to join! 😄

Notes will be taken by the host, and posted after the meeting. Notes are free to be edited through comments on the meeting notes at a later time.

**Comment by torch2424** [comment]
An Agenda Item I would like to bring up:

Wasm Summit! https://twitter.com/dassurma/status/1193958209859379201

Would love to hear if anyone has any plans to try and make it to it. I'll definitely be going. If you can't go, would love to try and share some ideas for AssemblyScript either by through a Talk, or just general chats I'll be having around the event.

**Comment by torch2424** [comment]
Also, let's investigate Google Hangouts over Zoom, just because it may provides the same features with less limitations for free users 😀

**Comment by torch2424** [comment]
Another Agenda Item from me:

More AsBind Stuff! Got it somewhat working, will try and get something nice to demo tommorrow 😄

**Comment by jayphelps** [comment]
Is the meeting still going on right now? I've requested to join the Hangout but haven't been accepted to enter. 😄

**Comment by MaxGraey** [comment]
@jayphelps we all can't connect to the room=)
@torch2424 where are you? Let us in! =)

**Comment by willemneal** [comment]
For those waiting we've moved here: https://meet.google.com/gik-ngxs-zfb

**Comment by willemneal** [comment]
[Here are the minutes from the meeting.](https://docs.google.com/document/d/1agQC2UcRcdsSSWL3CdtXoQ7YySGG8dhEOQafPBxHOK0/edit?usp=sharing)

**Comment by torch2424** [comment]
My apologies everyone, my phone died while I was sleeping, and my alarms never went off 😭 

Huge thank you to @willemneal for saving the meeting! 👏 and again, my deepest apologies!

**Comment by torch2424** [comment]
Again, huge apologies to everyone. I promise I'll get better at hosting these. I'll try and setup a buddy system perhaps and see if I can get someone to be a backup in the future. 

But what happened from me: Was up late last night preparing a [cool benchmark for asbind](https://github.com/AssemblyScript/meta/issues/17) to show off at the meeting. My phone wasn't fully plugged in, and died after I went to bed. So my alarms didn't go off, and I was extremely tired. Thus, me staying up late didn't even help 😂 😭 

And a huge thank you for everyone for there patience and letting the meeting continue! 👏 

Closing this issue, and opening the [next meeting issue](https://github.com/AssemblyScript/meta/issues/17). 😄
